### ISSUE TITLE
Think about dependabot

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#51]
A few thoughts:

- Security notifications from dependabot are very useful and we want them. I would also not mind dependabot opening a PR to fix those.
- I don't like how dependabot adds so much noise to the PR list. I think I would even prefer not using dependabot for that reason. I probably prefer a PR that includes all updates instead of one PR per dependency.
- The large number of PRs probably also leads to us not testing every single one. 
- However, I see that we somehow want to be reminded to regularly update our dependencies. Maybe there are some bots that can just regularly ping is on an issue?

**Comment by lkiesow** [comment]
Dependabot lets you control the frequency of updates. You can set them to weekly or monthly updates only.

**Comment by LukasKalbertodt** [comment]
Yeah true, but I still dislike having 20 or so PR every month :/ It's just mostly noise in the PR list IMO.

**Comment by lkiesow** [comment]
If we have reasonably good tests, we can also have dependabot merge them automatically if the tests pass.
Other options include setting the version update range  (control when pull requests are created in the first place) and only receiving security updates.
